src/Libraries/RevitNodes/Properties/Resources.en-US.resx

```diff
@@ -429,4 +429,10 @@
   <data name="DocumentNotWorkshared" xml:space="preserve">
     <value>Current Document is not workshared</value>
   </data>
+  <data name="NoElementsToPurge" xml:space="preserve">
+    <value>No Elements to purge in the current document</value>
+  </data>
+  <data name="PurgedElements" xml:space="preserve">
+    <value>Purged {0} elements, form the current document</value>
```

Comment:
```suggestion
    <value>Purged {0} elements from the current document.</value>
```

Reply:
actually not using this anymore. Deleting it from the resources.